packages/proxy/schema/index.ts

```diff
@@ -190,8 +181,34 @@ export const AvailableEndpointTypes: { [name: string]: ModelEndpointType[] } = {
   "NousResearch/Nous-Hermes-2-Yi-34B": ["together"],
   "deepseek-ai/deepseek-coder-33b-instruct": ["together"],
   "meta-llama/Llama-3.3-70B-Instruct-Turbo": ["together"],
+  "meta-llama/Llama-3.3-70B-Instruct-Turbo-Free": ["together"],
```

Comment:
whille you're at it, i've been meaning to move this bit of data into the model defintiion (it's super annoying to edit it out here). Do you want to take a crack at that?

Reply:
Let's do that in a follow-up? Would love to get this shipped and then do the refactor separately

Reply:
sgtm